src/urllib3/util/__init__.py

```diff
@@ -1,4 +1,5 @@
 # For backwards compatibility, provide imports that used to be here.
+from . import ssl_match_hostname
```

Comment:
We don't want to make this function available here as `urllib3.util` is public API.